webapp/packages/plugin-administration/src/ConfigurationWizard/Finish/FinishPage.tsx

```diff
@@ -6,54 +6,33 @@
  * you may not use this file except in compliance with the License.
  */
 import { observer } from 'mobx-react-lite';
-import styled, { css } from 'reshadow';
 
 import { ConfigurationWizardService } from '@cloudbeaver/core-administration';
-import { BASE_LAYOUT_GRID_STYLES, Button, useFocus, useStyles, useTranslate } from '@cloudbeaver/core-blocks';
+import { Button, s, useFocus, useS, useTranslate } from '@cloudbeaver/core-blocks';
 import { useController } from '@cloudbeaver/core-di';
 
-const styles = css`
-  layout-grid {
-    width: 100%;
-    flex: 1;
-  }
-
-  layout-grid-inner {
-    min-height: 100%;
-  }
-
-  layout-grid-cell {
-    composes: theme-background-surface theme-text-on-surface theme-border-color-background from global;
-    position: relative;
-    border: solid 1px;
-    padding: 16px 24px;
-  }
-
-  p {
-    line-height: 2;
-    white-space: pre;
-  }
-`;
+import styles from './FinishPage.m.css';
 
 export const FinishPage = observer(function FinishPage() {
   const translate = useTranslate();
   const service = useController(ConfigurationWizardService);
   const [focus] = useFocus<HTMLDivElement>({
     focusFirstChild: true,
   });
+  const style = useS(styles);
 
-  return styled(useStyles(BASE_LAYOUT_GRID_STYLES, styles))(
-    <layout-grid>
-      <layout-grid-inner>
-        <layout-grid-cell ref={focus} as="div" data-span="12">
+  return (
+    <div className={s(style, { layoutGrid: true })}>
```

Comment:
Please use Container ColoredContainer and Group components and style admin pages similar to Users admin page. Please don't just migrate styles especially if this styles if flex-box container usually you can replace this elements and styles with use of Containers combination